### PR TITLE
Jenkinsfile notarization

### PR DESCRIPTION
**Pull request by cadsit**

### CODE REVIEW

File: jenkins/Jenkinsfile [hunk 1]
```diff
@@ -477,6 +477,9 @@ timestamps {
                         }
 
                         stage ('Signing') {
+                            def notarize_mac = true
+                            def notarize_ios = false
```
Comment (rolfbjarne) [inline jenkins/Jenkinsfile]: You can enable iOS in the xcode11 branch.

**Comment by xamarin-release-manager** [comment]
❌ [Build was (probably) aborted](https://jenkins.internalx.com/job/macios/job/PR-6869/3/display/redirect)

🔥 [Jenkins job](https://jenkins.internalx.com/job/macios/job/PR-6869/3/display/redirect) (on internal Jenkins) failed in stage(s) 'Install Provisioning Profiles' 🔥 : hudson.AbortException: script returned exit code 104

✅ [Build succeeded](https://jenkins.internalx.com/job/macios/job/PR-6869/3//console)
✅ Packages: [xamarin.ios-12.99.3.27.pkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-6869/6e019bd4d66a4cb2722ab19c459f5466de1eface/3/package/xamarin.ios-12.99.3.27.pkg) [xamarin.mac-5.99.3.27.pkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-6869/6e019bd4d66a4cb2722ab19c459f5466de1eface/3/package/xamarin.mac-5.99.3.27.pkg)

**Comment by xamarin-release-manager** [comment]
❌ [Build was (probably) aborted](https://jenkins.internalx.com/job/macios/job/PR-6869/4/display/redirect)

🔥 [Jenkins job](https://jenkins.internalx.com/job/macios/job/PR-6869/4/display/redirect) (on internal Jenkins) failed in stage(s) 'Install Provisioning Profiles' 🔥 : hudson.AbortException: script returned exit code 104

✅ [Build succeeded](https://jenkins.internalx.com/job/macios/job/PR-6869/4//console)
✅ Packages: [xamarin.ios-12.99.3.28.pkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-6869/8129cb9b7b71488b4615f696ebc25af423a8de20/4/package/xamarin.ios-12.99.3.28.pkg) [xamarin.mac-5.99.3.28.pkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-6869/8129cb9b7b71488b4615f696ebc25af423a8de20/4/package/xamarin.mac-5.99.3.28.pkg)

**Comment by cadsit** [comment]
@rolfbjarne should we point the existing PKG statuses to the notarized copies and have a separate one for the non-notarized PKGs?

**Comment by rolfbjarne** [comment]
@cadsit that's a good question, and I'm not sure. It feels like the safe option would be to create a new PKG status for the notarized packages, so that might be the best option for now. It'll be easy enough to switch later.

**Comment by cadsit** [comment]
@rolfbjarne should this line be updated with the notarized uris? https://github.com/xamarin/xamarin-macios/blob/jenkinsfile-notarization/jenkins/Jenkinsfile#L661

**Comment by xamarin-release-manager** [comment]
❌ [Build was (probably) aborted](https://jenkins.internalx.com/job/macios/job/PR-6869/5/display/redirect)

🔥 [Jenkins job](https://jenkins.internalx.com/job/macios/job/PR-6869/5/display/redirect) (on internal Jenkins) failed in stage(s) 'Build' 🔥 : org.jenkinsci.plugins.workflow.steps.FlowInterruptedException

**Comment by rolfbjarne** [comment]
@cadsit 

> @rolfbjarne should this line be updated with the notarized uris? https://github.com/xamarin/xamarin-macios/blob/jenkinsfile-notarization/jenkins/Jenkinsfile#L661

Eventually maybe, but that can wait.

**Comment by xamarin-release-manager** [comment]
❌ [Build was (probably) aborted](https://jenkins.internalx.com/job/macios/job/PR-6869/6/display/redirect)

🔥 [Jenkins job](https://jenkins.internalx.com/job/macios/job/PR-6869/6/display/redirect) (on internal Jenkins) failed in stage(s) 'Build' 🔥 : hudson.AbortException: script returned exit code 2

🔥 [Build failed](https://jenkins.internalx.com/job/macios/job/PR-6869/6//console) 🔥

**Comment by xamarin-release-manager** [comment]
❌ [Build was (probably) aborted](https://jenkins.internalx.com/job/macios/job/PR-6869/7/display/redirect)

🔥 [Jenkins job](https://jenkins.internalx.com/job/macios/job/PR-6869/7/display/redirect) (on internal Jenkins) failed in stage(s) 'Install Provisioning Profiles' 🔥 : hudson.AbortException: script returned exit code 104

✅ [Build succeeded](https://jenkins.internalx.com/job/macios/job/PR-6869/7//console)
✅ Packages: [xamarin.ios-12.99.3.30.pkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-6869/2acfed275f9d9edfa8b0fdbba32bf21c0077ca08/7/package/xamarin.ios-12.99.3.30.pkg) [xamarin.mac-5.99.3.30.pkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-6869/2acfed275f9d9edfa8b0fdbba32bf21c0077ca08/7/package/xamarin.mac-5.99.3.30.pkg)[xamarin.ios-12.99.3.30.pkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-6869/2acfed275f9d9edfa8b0fdbba32bf21c0077ca08/7/package/xamarin.ios-12.99.3.30.pkg) [xamarin.mac-5.99.3.30.pkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-6869/2acfed275f9d9edfa8b0fdbba32bf21c0077ca08/7/package/xamarin.mac-5.99.3.30.pkg)

**Comment by xamarin-release-manager** [comment]
❌ [Build was (probably) aborted](https://jenkins.internalx.com/job/macios/job/PR-6869/8/display/redirect)

🔥 [Jenkins job](https://jenkins.internalx.com/job/macios/job/PR-6869/8/display/redirect) (on internal Jenkins) failed in stage(s) 'Install Provisioning Profiles' 🔥 : hudson.AbortException: script returned exit code 104

✅ [Build succeeded](https://jenkins.internalx.com/job/macios/job/PR-6869/8//console)
✅ Packages: [xamarin.ios-12.99.3.31.pkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-6869/d1e2377cf4a1476429d5632758c37093b2a2b1e4/8/package/xamarin.ios-12.99.3.31.pkg) [xamarin.mac-5.99.3.31.pkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-6869/d1e2377cf4a1476429d5632758c37093b2a2b1e4/8/package/xamarin.mac-5.99.3.31.pkg)[xamarin.ios-12.99.3.31.pkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-6869/d1e2377cf4a1476429d5632758c37093b2a2b1e4/8/package/notarized/xamarin.ios-12.99.3.31.pkg) [xamarin.mac-5.99.3.31.pkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-6869/d1e2377cf4a1476429d5632758c37093b2a2b1e4/8/package/notarized/xamarin.mac-5.99.3.31.pkg)

**Comment by xamarin-release-manager** [comment]
❌ [Build was (probably) aborted](https://jenkins.internalx.com/job/macios/job/PR-6869/9/display/redirect)

🔥 [Jenkins job](https://jenkins.internalx.com/job/macios/job/PR-6869/9/display/redirect) (on internal Jenkins) failed in stage(s) 'Install Provisioning Profiles' 🔥 : hudson.AbortException: script returned exit code 104

✅ [Build succeeded](https://jenkins.internalx.com/job/macios/job/PR-6869/9//console)
✅ Packages: 
* [xamarin.ios-12.99.3.32.pkg (Not notarized)](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-6869/745bd21039cd3188f2b1e7bc4706389a1143e283/9/package/xamarin.ios-12.99.3.32.pkg)
* [xamarin.mac-5.99.3.32.pkg (Not notarized)](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-6869/745bd21039cd3188f2b1e7bc4706389a1143e283/9/package/xamarin.mac-5.99.3.32.pkg)
* [xamarin.ios-12.99.3.32.pkg (Notarized)](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-6869/745bd21039cd3188f2b1e7bc4706389a1143e283/9/package/notarized/xamarin.ios-12.99.3.32.pkg)
* [xamarin.mac-5.99.3.32.pkg (Notarized)](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-6869/745bd21039cd3188f2b1e7bc4706389a1143e283/9/package/notarized/xamarin.mac-5.99.3.32.pkg)

**Comment by cadsit** [comment]
@rolfbjarne good to go?

**Pull request merged**